pkg/util/k8sutil/pod_util.go

```diff
@@ -127,6 +127,9 @@ func applyPodPolicy(clusterName string, pod *v1.Pod, policy *spec.PodPolicy) {
 	if len(policy.Tolerations) != 0 {
 		pod.Spec.Tolerations = policy.Tolerations
 	}
+	if !*policy.AutomountServiceAccountToken {
```

Comment:
What if `AutomountServiceAccountToken` is nil?

Reply:
Good call. I actually like this better. Now, I just check for nil, so that if you explicitly set it as true, it still shows up in the manifest. Before it wouldn't and just rely on Kubernetes' default.